package.json

```diff
@@ -29,13 +29,13 @@
     "@babel/core": "^7.16.7",
     "@emotion/css": "^11.1.3",
     "@grafana/aws-sdk": "0.0.42",
-    "@grafana/data": "8.5.10",
```

Comment:
I'm curious what is the motivation in updating these packages? Is it needed for some other changes we want to add?

Reply:
Yup! It's preparation for adding getDefaultQuery from 9.3.2 [here:](https://github.com/grafana/cloud-data-sources/issues/79) I forgot to add it in the description, will do it now!